3.9-minimal/Dockerfile.rhel8

```diff
@@ -1,19 +1,25 @@
-# This image provides a Python 3.9 environment you can use to run your Python
-# applications.
-FROM ubi8/s2i-base:1
+FROM ubi8/ubi-minimal:latest
 
 EXPOSE 8080
 
 ENV PYTHON_VERSION=3.9 \
-    PATH=$HOME/.local/bin/:$PATH \
+    # /opt/app-root/bin - the main venv \
+    # /opt/app-root/src/bin - app-specific binaries \
+    # /opt/app-root/src/.local/bin - tools like pipenv \
+    PATH=/opt/app-root/bin:/opt/app-root/src/bin:/opt/app-root/src/.local/bin:$PATH \
     PYTHONUNBUFFERED=1 \
     PYTHONIOENCODING=UTF-8 \
     LC_ALL=en_US.UTF-8 \
     LANG=en_US.UTF-8 \
     CNB_STACK_ID=com.redhat.stacks.ubi8-python-39 \
     CNB_USER_ID=1001 \
     CNB_GROUP_ID=0 \
-    PIP_NO_CACHE_DIR=off
+    PIP_NO_CACHE_DIR=off \
+    # The following variables are usually available from parent s2i images \
+    STI_SCRIPTS_PATH=/usr/libexec/s2i \
+    APP_ROOT=/opt/app-root \
+    HOME=/opt/app-root/src \
+    PLATFORM="el8"
```

Comment:
Just a nitpick thought: It might be good to define the commn s2i variables on top and use them in the PATH definition.

Reply:
@frenzymadness What do you think?

Reply:
Implemented in the latest commit.